### ISSUE TITLE
About the Sources

### ISSUE THREAD

**Issue by serapath** [nicksellen/ponderings#1]
All the sources you listed are basically crap, except for one - the bank of england.

Most videos and sources have opinions on the matter, go to unnecessary length, have a bias in them, because they want to simplify things.
I personally have some sympathy for "positivemoney", but even then, try to stay with the SPEC.
The SPEC is the central banks and the rules surrounding them and try to interpret them.

In the end, irrespective of the history and origins of money, what we have today is determined by "double entry accounting" and the rules around how banks and companies do that activity. Everything else is very likely to lead to false conclusions and you have wildly varying points of view that can consume a lot of time without contributing anything to an understanding.

(btw. i dislike the "moneyless manifesto" because of the fact they are not giving it for free - thats a scam!)

**Comment by serapath** [comment]
So I recommend to go with:
- http://www.bankofengland.co.uk/publications/Documents/quarterlybulletin/2014/qb14q1prereleasemoneyintro.pdf
- https://www.youtube.com/watch?v=ziTE32hiWdk
- http://www.bankofengland.co.uk/publications/Documents/quarterlybulletin/2014/qb14q1prereleasemoneycreation.pdf

You listed all of them. Throw away the other crap, because it's just interpretation of the truth - which is the SPEC - which are the rules that banks have to adhere to and the software and networks that connect banks are made to comply to.

To figure it out that way requires a little bit more brain computation, but will save you a lot of time and confusion I think.

**Comment by nicksellen** [comment]
Thanks for the input :)

Have you got a link for SPEC - it's not a very googleable term and I can't seem to find anything about it..... it sounds a very useful thing to look at though.

On resources being _crap_ or just _interpretations of the truth_ - I'm afraid _everything_ is an interpretation of the truth, and anything might be considered crap by someone else - including your comments, how do I know to trust you on what to throw away or not? And even if I read the SPEC, how do I know that's what banks _actually_ do?

My approach is to try and _understand everything_ - not only how banking works, but how people think it works too. Sometimes people come at it from a totally different perspective and this might be insightful (even if they are factually incorrect about some stuff). I am not corrupted from reading all the viewpoints, and it also helps when trying to educate people (if I can become knowledgeable enough on the topic) - as you need to have some understanding of their starting point and what the journey of learning is going to look like, not just present the conclusions.

Also, remember the topic is not _understanding how banking works_ - I don't have a specific description of what is _is_ about, but the starting point is more about _the morality of money_, and considerations by people like Michael Sandel such as "what money cannot buy" are wider philosophical or moral concerns not directly about the international banking system. I'm thinking of actually widening the topic to just "ponderings" and writing about _everything_ I think about as I'm finding it harder and harder to separate considerations about money from everything else.

If you have more resources or perspectives please let me know though, I will try and digest everything thrown at me in the fullness of time :)

**Comment by serapath** [comment]
No not everything is an interpretation of the truth.
There is a Javascript Specification and that Specification is what implementations fullfill (even though they might differ in some minor aspects)
The same is true for central banking, they have strict rules which define the "legal moves" possible within the financial accounting system that powers banks and they have very strict protocols.

The spec itself doesnt tell you what "playing the game within that rules" means, because its neutral.
In order to form an opinion or judgement, you need to interpret them and thats where all these "colored videos" might be interesting, but its kind of pointless if you dont know the rules upfront.

I would suggest starting with wikipedia, because extracting something meaningful from the documents on the website of the ECB, BoE, the FED or other central banks is more cumbersome.

http://en.wikipedia.org/wiki/Money_creation
http://en.wikipedia.org/wiki/Fractional-reserve_banking

But first check: http://en.wikipedia.org/wiki/Balance_sheet

If you cannot read **Balance Sheets** you will never be able to understand central banking.
Balance Sheets are a "snapshot" of what is formally or bureaucratically accepted as reality.
Balance Sheet (t+1) = Balance Sheet (t) + Profit/Loss Statement (t)

Apply all the "flows" in period t to the "current state" at the beginning of period t and it will result in the "new current state" at period t+1.

Balance Sheets are super simple, maybe you already know them.
You should then make yourself familiar with **Accounting Records**, how to book transactions on ledgers. There are 4 fundamental version of them.
1. One that enlarges both sides of the balance sheet
2. One that shrinks both sides of the balance sheet
3. One that changes the order of positions on the left side of the balance sheet
4. One that changes the order of the positions on the right side of the balance sheet

Every transaction, that involves "money" touches at least 2 balance sheets.
Every transaction, that involves "cash" touches at least 3 balance sheets.

On the left side of a balance sheets you have all the so called assets.
On the right side you see who owns what (e.g. either as a liability or as equity)

You can devide the whole economy into
1. central bank(s)
2. private banks
3. government
4. private households
5. businesses

(you can even simplify and lumpsum 4. and 5. into the same bucket)
I can send you an excelsheet that pictures the relations.

**Comment by serapath** [comment]
You have to become familiar with those formalisms, otherwise you will turn in circles.

**Comment by nicksellen** [comment]
Oh, SPEC isn't an actual thing, it's just a name you were giving to "the rules/laws governing banking"?

> No not everything is an interpretation of the truth

I mean this in the deeply philosophical sense, how do I know anything at all? So when you say:

```
The same is true for central banking, they have strict rules which define the "legal moves" possible within the financial accounting system that powers banks and they have very strict protocols.
```

I don't _know_ if any of it is true. I _believe_ you that they have strict rules, I _believe_ they are strict. I don't _know_ how well they are followed. I have not observed them. I haven't even seen the rules. I know almost nothing. So _from my perspective_ I am swimming in a sea of interpretations of truth, everything I come across I must be skeptical about and question how whether _they_ know that, and _how_ they know that.

What would happen if somebody came along tomorrow and told me with strong certainty that the banks don't follow any rules at all. I would be skeptical. But why? It doesn't fit the model I am building up so far. But I still haven't observed any of this myself. It is just peoples words so far. They are models of reality. How do I come to have a model of reality that is closest to reality? Who do I believe? Who do I trust? I must be ready and willing to throw any existing models of reality I have out the window.

I can't answer any of that. I have no idea who to trust. So I trust everyone and no-one! As I listen to each model of reality described to me I build up my own model. If two models clash with each other I try and think of tests that could resolve the discrepancy and dig deeper. If I read 10 accounts of the same thing from 10 perspectives and they agree then this is good. If I get 10 different versions, then there is more to dig into.

There is clearly a whole bunch of stuff that is more factual here, the basic mechanics of the process etc, so thanks for the links and extra concepts to learn, I will read up on all that so I can re-read what you wrote without my head spinning!

**Comment by nicksellen** [comment]
> (btw. i dislike the "moneyless manifesto" because of the fact they are not giving it for free - thats a scam!) 

http://www.moneylessmanifesto.org/book/foreword-by-charles-eisenstein/ is the book, the whole thing, for free.

The justification for charging for the print edition is:

```
If you do buy a normal paperback copy of this book, your money is going towards two things – the cost of printing a book in a money-based economy, which is a reality of our Age; and supporting some other work – that of Permanent Publications in the field of Permaculture, and that of myself and others in our efforts to create a fully localised (and land-based) gift economy in the UK
```

(from http://www.moneylessmanifesto.org/why-free/)

So it's the argument that it's a _transitional reality_ of life right now if you want to have a book physically printed it's gunna cost money. In the way they can, they provide the content of the book entirely for free.

I don't feel intuitively in my current position that moneyless is the answer to our worldly problems, but I want to be _open_ to it, especially before having read it; I simply don't know enough about the point they are making yet.

**Comment by pauldavisthefirst** [comment]
I started reading the moneyless manifesto, and I found it sorely lacking. Mostly because I got several sections in without being able to find any evidence that the author had really thought about what money is, distinct from what a monetary culture might be.

In both barter and gift economies, your ability to exchange services and goods with others is limited by your ability to find people who will barter or gift you the things you want or need.

At its heart, money is a way of removing this requirement, which at some point must have been immensely liberating. It allows you to exchange goods and services with others, regardless of whether or not you have anything they might want at that time. 

It also leads to the creation of much larger webs of dependency. If I get eggs from someone by trading them leather, or if someone simply gifts me a service or supply that I need, then the dependency relationship extends only to the two of us. But if I buy eggs with money, then both I and the egg-seller become part of a much larger web of interconnection and interdependence. This web, however, is largely anonymous, and there's a convincing case to be made for this being a not particularly postive thing.

At its heart, money isn't really any more or less than this: a means of decoupling ability to trade from having the appropriate relationship with the traders (knowing them, them be willing to trade with you, them being close enough to you, and so forth). In this sense, money is a liberation technology.

However, I did mention David Graber's book on Debt, which claims that social debt relationships predate the existence of money. This is important because it suggests that there are "financial" relationships that predate whatever kinds of relationships were enabled by money, and further, suggests that our use of money was shaped by them, and they were affected by money. 

A moneyless world is definitely a cozier more personal world than the one we live in now. But that is because it is a smaller, massively more limited world in which prejudice, geographical location, and other inequitable conditions will play a substantive role in determining one's quality of life.

**Comment by nicksellen** [comment]
@pauldavisthefirst yes this social debt thing (my friend Joe had read the book so filled me in a bit on its innards) seems to be claimed to be the true precursor to money (barter being exaggerated, or even a created myth). I haven't read it yet though. It's interesting to me that social debt AND barter are still actually alive and well alongside money (at least social debt is very common still, I have a bag of lentils next to me right now on that basis).

The decoupling power of money intuitively makes sense to me and seems useful. Do you know what the moneyless counter argument actually is specifically? Someones gunna have to read this thing!

**Comment by nicksellen** [comment]
@pauldavisthefirst https://github.com/nicksellen/ponderings/issues/2 has the specific moneyless issue split out.

**Comment by serapath** [comment]
The key to money how it works in our time is understanding balance sheets and accounting.
If you don't start there, you cannot understand money because everything what you read about money will be mixed with thoughts about how people imagine money or would like money to be.

Understanding balance sheets and transactions and how to record them and how money is created, transfered and destroyed enables you to write "source code" or read "source code" written by others.

Now, depending on what you want to do, some transactions are legal and some are not. Some are common and some are impossible or unlikely.

Its totally possible to argue about money, but if you don't understand the language of money, i think you would be able to improve your understanding of money a lot. But feel free to try whatever approach suits you :-)

**Comment by nicksellen** [comment]
> everything what you read about money will be mixed with thoughts about how people imagine money

Money is _entirely_ just imagined though :) - it only has value because we all (or most of us) believe it does. So I don't think "how people imagine money" can be entirely excluded from the debate.

But if you're just referring to the occasion when people are just _wrong_ about a factual matter, then I hope to be able to learn enough to detect this. I can only start by assuming everybody is wrong and build from there, using my generalised mechanisms of detecting falsehoods.

**Comment by serapath** [comment]
You can drive a car and imagine it is powered by magic even though it works different.
People observe money the way they experience it and draw conclusions from that, that might be useful for predicting the effects of money on society or economy or not.
People experience it differently (some have plenty of money, some have none) and form different opinions which they cannot resolve, because they do not go to the source in order to come to a common understanding in which both of their perspectives are covered by a coherent model.

**Comment by nicksellen** [comment]
What about the people that _wrote_ the source? Do they not have some ideology/morality/perspective/bias when writing the rules? What is the _root_ source of truth from which the rules are born?

**Comment by serapath** [comment]
That is true, but I think a nice approach would be to first discover those rules and once familiar with them (because they describe the status quo) it would be an interesting thing to speculate about ideology/morality/perspective/bias :-)

**Comment by nicksellen** [comment]
Ok, now just have to wait 3 years for me to read lots of words and numbers
:)

**Comment by serapath** [comment]
No, you can learn that in mere days maybe weeks. It's dead simple.
https://www.youtube.com/watch?v=ixCPM5HznRU
There are tons of other videos. If you try to figure it out with the same rigor you use when it comes to programming stuff, you'll know it all very fast.
On Khan Academy they have some nice series about accounting and balance sheets and even about banks and accounting.
e.g. https://www.youtube.com/watch?v=MILF-9GeMDQ
